recipes/doxygen/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 2.8.12)
+cmake_policy(SET CMP0057 NEW) # if IN_LIST
```

Comment:
Is it really required? According the documentation, it requires Cmake 3.3

Reply:
I've changed it to 3.3 and removed the policy